### PR TITLE
Push on condition

### PR DESCRIPTION
**Pull request by EWhite613**
#patch#

# Changelog
- Add options that allows CI to only push when `push-var` and `push-on` are equal

### CODE REVIEW

**Comment by ember-cli-visual-acceptance** [comment]
# Visual Acceptance tests:
### No changes

File: index.js [hunk 1]
```diff
@@ -554,9 +565,11 @@ module.exports = {
                return runCommand('git', ['add', '-f', '-A', options.imageDirectory], true).then(function (params) {
                  console.log('Git commit')
                  return runCommand('git', ['commit', '-m',
-                  '"Adding new baseline images [ci skip]"'], true).then(function (params) {
-                    console.log('Git push')
-                    return runCommand('git', ['push', 'origin', 'HEAD:' + options.branch], true)
+                  'Adding new baseline images [ci skip]'], true).then(function (params) {
+                    if (options.pushConditionCompare === options.pushConditionMatch) {
+                      console.log('Git push')
```
Comment (sglanzer) [inline index.js]: Log?

File: index.js [hunk 2]
```diff
@@ -657,9 +681,11 @@ module.exports = {
...
```
Comment (sglanzer) [inline index.js]: Ditto

File: index.js [hunk 3]
```diff
@@ -500,6 +500,17 @@ module.exports = {
           type: String,
           default: 'master',
           description: 'branch to push to'
+        }, {
+          name: 'push-condition-compare',
```
Comment (sglanzer) [inline index.js]: It's not totally clear which of these is supposed to be coming from the CI system and which is configured ahead of time, can they be worded for more clarity?

**Comment by sglanzer** [comment]
Approved

**Comment by sglanzer** [comment]
Approved - thanks for the change

**Pull request merged**